packages/sensors_plus/sensors_plus/example/lib/main.dart

```diff
@@ -144,6 +154,18 @@ class _MyHomePageState extends State<MyHomePage> {
             _accelerometerValues = <double>[event.x, event.y, event.z];
           });
         },
+        onError: (e) {
+          showDialog(
+              context: context,
+              builder: (context) {
+                return const AlertDialog(
```

Comment:
I think this choice be left to the develop, how they want to handle it and what UI they want to show, we should not add hardcoded UI

Reply:
You probably confusing the package with example app. In example app we can use whatever we want.